website/examples/date-picker-dialog.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import { DayPicker } from 'react-day-picker';
+import { usePopper } from 'react-popper';
+
+import { format, isValid, parse } from 'date-fns';
+import FocusTrap from 'focus-trap-react';
+
+export default function Example() {
+  const [selected, setSelected] = React.useState<Date>();
+  const [inputValue, setInputValue] = React.useState<string>('');
+  const [isPopperOpen, setIsPopperOpen] = React.useState(false);
+
+  const popperReference = React.useRef<HTMLDivElement>(null);
+  const buttonRef = React.useRef<HTMLButtonElement>(null);
+  const [popperElement, setPopperElement] =
+    React.useState<HTMLDivElement | null>(null);
+
+  const closePopper = () => {
+    setIsPopperOpen(false);
+    buttonRef?.current?.focus();
+  };
+
+  const popper = usePopper(
+    popperReference.current,
+    popperElement,
+    {
+      placement: 'bottom-start'
+    }
+  );
+
+  const formatDate = (date: Date) => format(date, 'P');
```

Comment:
I think this wrapper adds clutter in the example code. We should always use `format` from date-fns. We could replace the calls to `formatDate` with `format(date, formatString)`, where `formatString` is declared at the top of the function (see my other comment).

Reply:
Done